app/ui/src/app/dashboard/connections.component.html

```diff
@@ -32,7 +32,7 @@ <h1>Create a Connection</h1>
 
               <div class="card-pf-top-element">
                 <span class="card-pf-icon-large image-icon">
-                <img src="../../../assets/icons/{{ connection.connectorId }}.connection.png">
+                <img src="{{ connectionIcon(connection) }}">
```

Comment:
Yeah, definitely work looking into updating the store.  Like we could add an override to this class:

https://github.com/syndesisio/syndesis/blob/master/app/ui/src/app/store/connection/connection.service.ts

That basically sets the icon, you'd override this function:

https://github.com/syndesisio/syndesis/blob/master/app/ui/src/app/store/entity/entity.store.ts#L60-L75

and probably modify this too:

https://github.com/syndesisio/syndesis/blob/master/app/ui/src/app/store/entity/entity.store.ts#L77-L86

to update/enhance the `icon` property on the connection/connector.  Probably need to do it for both the connection and the connector stores.